src/hooks/useClickOutside/useClickOutside.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useRef } from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { useOnClickOutside } from '.';
+
```

Comment:
In this case it might not even matter since you use the mockFn scoped inside the test but make sure you always reset your mocks by adding a jest.RestoreAllMocks() in a beforeEach or afterEach.

This would also probably allow you to move the mockFN and the TestComponent outside of the tests (to re-use them between tests). Although in this particular case I don't think it would benefit you a whole lot.